client/dashboard/emails/utils/is-titan-monthly-product.ts

```diff
@@ -0,0 +1,6 @@
+import { Product } from '@automattic/api-core';
+import { TITAN_MAIL_MONTHLY_SLUG } from '../constants';
+
+export function isTitanMonthlyProduct( titanMailProduct: Product ): boolean {
```

Comment:
Taken from https://github.com/Automattic/wp-calypso/blob/127f23fb25d8dcad4aa2c8ecba220c7522bff6ff/client/lib/titan/is-titan-monthly-product.ts#L4